Pod/Classes/Core/RNDynamicScreenDimensions.h

```diff
@@ -0,0 +1,6 @@
+#import <React/RCTBridgeModule.h>
+#import <React/RCTEventEmitter.h>
+
+@interface RNDynamicScreenDimensions : RCTEventEmitter <RCTBridgeModule>
```

Comment:
@ds300 We should not be using somebody else’s namespace. I.e. this needs to be prefixed with `AR`, but we can do that in follow-up.